codegenerator/cli/templates/static/codegen/src/eventFetching/hyperfuel/HyperFuel.res

```diff
@@ -102,8 +101,10 @@ module LogsQuery = {
           Data,
           ReceiptIndex,
           ReceiptType,
-          Ra,
           Rb,
+          // TODO: Include them only when there's a mint/burn receipt selection 
+          SubId,
+          Val
```

Comment:
Does it affect the response type? Probably not a big deal to include otherwise.

Reply:
It just feels a little bit wasteful to request irrelevant data when there are only log events. Although, these fields are null in this case, so not a huge response size increase.